tests/unit/test_buffer_handling.py

```diff
@@ -13,6 +13,10 @@
 from power_grid_model._core.dataset_definitions import ComponentType, DatasetType
 from power_grid_model._core.power_grid_meta import initialize_array, power_grid_meta_data
 
+batch_ndim = 2
+scenario_total_elements = 4
+batch_total_elements = 8
```

Comment:
here and elsewhere: module-level constants shall be capitalized according to https://peps.python.org/pep-0008/#constants